components/greaselion/browser/greaselion_service.h

```diff
@@ -60,7 +60,7 @@ class GreaselionService : public KeyedService,
 
  private:
   friend class ::GreaselionServiceTest;
-  virtual void SetBrowserVersionForTesting(const base::Version& version);
+  virtual void SetBrowserVersionForTesting(const base::Version& version) = 0;
```

Comment:
Link phase was failing in debug mode on Windows without this.

Reply:
Thanks for finding that